examples/quickstart/main.rs

```diff
@@ -153,6 +158,7 @@ async fn main() {
     let tx_signer = EthereumWallet::from(wallet.clone());
 
     let provider = ProviderBuilder::new()
+        .with_chain(NamedChain::Base)
```

Comment:
Pain #1: I was using `with_chain_id` instead of `with_chain`.
Don't do that. It's bad.

Reply:
you will still make a correspondence between Chain and NamedChain right? I hate how many Chain objects we have lying around

Reply:
yes